### PR TITLE
Sources available for user's groups.

### PR DESCRIPTION
**Pull request by idegtiarov**
Add filtering for the user's allowed content source based on user's groups.
The default group is added to every newly created user and should contain default content sources available to all users.

The JIRA related ticket is https://vpalresearch.atlassian.net/browse/AD-259

### CODE REVIEW

File: bridge_adaptivity/bridge_lti/admin.py [hunk 1]
```diff
@@ -1,9 +1,66 @@
...
+from django.contrib.auth.models import Group
+from django.forms import ModelForm, ModelMultipleChoiceField
 from django.utils.translation import ugettext_lazy as _
 
 from .models import BridgeUser, LtiConsumer, LtiProvider, LtiUser, OutcomeService
 
+admin.site.unregister(Group)
+
+
+class GroupForm(ModelForm):
```
Comment (AndreyLykhoman) [inline bridge_adaptivity/bridge_lti/admin.py]: Please add a docstring.

File: bridge_adaptivity/api/backends/api_client.py [hunk 2]
```diff
@@ -143,16 +146,18 @@ def apply_data_filter(data, filters=None, **kwargs):
...
 
     LTI Providers which expose courses content blocks to use in adaptive collections.
+    :param request: Request object
     :param source_ids: LtiConsumer ID or list of the LtiConsumer ID or None
     :return: content_sources queryset
     """
     q_kw = {
-        'is_active': True
+        'is_active': True,
+        'available_in_groups__in': list(request.user.groups.all()),
```
[the remark this answers is not in the document]
Comment (idegtiarov) [reply bridge_adaptivity/api/backends/api_client.py]: if statement won't work in the current case, a request is required parameter.

File: bridge_adaptivity/api/backends/api_client.py [hunk 3]
```diff
@@ -143,16 +146,18 @@ def apply_data_filter(data, filters=None, **kwargs):
     return filtered_data
 
 
-def get_content_providers(source_ids=None):
+def get_content_providers(request=None, source_ids=None):
```
[the remark this answers is not in the document]
Comment (idegtiarov) [reply bridge_adaptivity/api/backends/api_client.py]: done

File: bridge_adaptivity/api/tests/test_views.py [hunk 4]
```diff
@@ -61,11 +65,11 @@ def test_base_api_client_calls(
 
         We use the id of the content source equal to 5 because in fixture this source has type `base`
         """
-        get_available_blocks(5)
+        get_available_blocks(self.request, 5)
```
[the remark this answers is not in the document]
Comment (idegtiarov) [reply bridge_adaptivity/api/tests/test_views.py]: NOt magic numbers but pk keys, which are comming from the fixtures. Will add comments.

File: bridge_adaptivity/bridge_lti/admin.py (hunk 1 above)
Comment (idegtiarov) [reply bridge_adaptivity/bridge_lti/admin.py]: as you wish!

**Pull request merged**